tests/test_base.py

```diff
@@ -139,6 +139,9 @@ def test_get_urls(urls):
     assert urls == ["http://hdl.handle.net/1885/10005"]
 
 
+@pytest.mark.skip(
+    'This should be refactored, the url does not respond anymore.'
+)
 def test_find_direct_links(direct_links):
     """Test direct link recognising"""
     assert direct_links == []
```

Comment:
I kind of fixed this in my PR #48 (actually I refactored everything in the tests)